fem/libceed/ceed.hpp

```diff
@@ -127,13 +95,14 @@ struct CeedData
       CeedVectorDestroy(&u);
```

Comment:
Also `CeedQFunctionContext build_ctx` needs to be destroyed or there will be a memory leak

Reply:
done.